### PR TITLE
Accessibility fixes

### PR DESCRIPTION
**Pull request by liamjohnston**

### CODE REVIEW

File: grow-website-gatsby/src/pages/index.tsx [hunk 1]
```diff
@@ -51,11 +52,15 @@ export const pageQuery = graphql`
...
+      <Fragment>
+        <Helmet>
+          <html lang="en-NZ" />
+          <title>Grow &endash; Empowering women to thrive in tech</title>
+        </Helmet>
+        <header className={styles.gutters}>
           <div className={styles.header}>
             <div className={styles.headerLogo}>
-              <img src={growLogo} alt="Grow" />
+              <img src={growLogo} alt="The Grow team" />
```
Comment (catepalmer) [inline grow-website-gatsby/src/pages/index.tsx]: I think we want this alt text on the banner image rather than the logo.

**Review by catepalmer — CHANGES_REQUESTED** [review]
Just one little change requested, otherwise looks good!

File: grow-website-gatsby/src/pages/index.tsx (hunk 1 above)
Comment (liamjohnston) [reply grow-website-gatsby/src/pages/index.tsx]: Whoops, yes that was the intent! 😬

Comment (liamjohnston) [reply grow-website-gatsby/src/pages/index.tsx]: @catepalmer Fixed :)

**Comment by liamjohnston** [comment]
🎉 🎉 🎉 

Screen Shot 2019-10-18 at 2 48 01 PM

**Review by catepalmer — APPROVED** [review]

**Pull request merged**